### PR TITLE
Event users are now included in user export

### LINKED ISSUE #786: Download passwords should download event users
Currently only downloads division users

### PR DESCRIPTION
**Pull request by itamaroryan**
Fixes #786

### CODE REVIEW

File: libs/database/src/lib/crud/users.ts [hunk 1]
```diff
@@ -34,6 +34,10 @@ export const getUserWithCredentials = (filter: Filter<User>) => {
   return db.collection<User>('users').findOne(filter);
 };
 
+export const getUsersWithCredentials = (filter: Filter<User>) => {
```
Comment (johnmeshulam) [inline libs/database/src/lib/crud/users.ts:37]: How is this "with credentials"? It looks like a regular users call

File: apps/backend/src/routers/api/admin/divisions/users.ts [hunk 2]
```diff
@@ -15,7 +15,13 @@ router.get('/', (req: Request, res: Response) => {
 router.get(
   '/export',
   asyncHandler(async (req: Request, res: Response) => {
-    const users = await db.getDivisionUsersWithCredentials(new ObjectId(req.params.divisionId));
+    const divisionUsers = await db.getDivisionUsersWithCredentials(
+      new ObjectId(req.params.divisionId)
+    );
+    const eventUsers = await db.getUsersWithCredentials({
+      assignedDivisions: new ObjectId(req.params.divisionId)
```
Comment (johnmeshulam) [inline apps/backend/src/routers/api/admin/divisions/users.ts:22]: Assigned divisions is an array - should the filter be a contains function?

File: libs/database/src/lib/crud/users.ts (hunk 1 above)
Comment (itamaroryan) [reply libs/database/src/lib/crud/users.ts:37]: Users usually return with credentials, it is when we want to return a safe user to the backend the we omit the "with credentials" and do some more logic (this follows similar functions in the same file)

File: apps/backend/src/routers/api/admin/divisions/users.ts (hunk 2 above)
Comment (itamaroryan) [reply apps/backend/src/routers/api/admin/divisions/users.ts:22]: Turns out that mongodb doesn't have a $include function for querying, yet this syntax does is a shorthand for the same thing, checking if assignedDivisions includes the relevant value. Should we maybe add a comment explaining this?

Comment (johnmeshulam) [reply apps/backend/src/routers/api/admin/divisions/users.ts:22]: Standard mongo knowledge so no need

**Review by johnmeshulam — APPROVED** [review]

**Pull request merged**